taxonomies/ingredients.txt

```diff
@@ -62808,8 +62918,13 @@ nl:voorjaarshoning
 # ingredient/fr:miel-de-printemps has 4 products in french @2019-05-28
 
 <en:honey
```

Comment:
isn't propolis different than honey? if it is, honey should not be a parent. We use parent only for the "is a" relationship. e.g. "green bell pepper" is a "bell pepper".

Reply:
yes it is. It's a product of bees but how to classify it?

Reply:
just a separate entry. We only make parents, when they also exist on products. This in order to keep the ingredients taxonomy small, and we would otherwise make a copy of the categories taxonomy.

Reply:
so propolis does not seem to be honey, it is a product of bee like honey, and royal jelly

Reply:
ok, done